README.md

```diff
@@ -124,6 +124,27 @@ ssl_certificate_key /path/to/certificate-key.pem;
 
 Also, `proto=https` option is needed for the `listen` directive.
 
+#### Self-signed certificate genration
+
+In case of using self-signed certificate with Tempesta, it's usefull
+to use openSSL to generate the key and certificate. Use the following
+shell command:
+
+~~~
+openssl req  -nodes -new -x509  -keyout tfw-root.key -out tfw-root.crt
+~~~
+
+You'll be prompted to fill several X.509 certifcate fields same for
+the subject and issuer. Use any valid values as you like.
+
+Now, `tfw-root.key` contains private key and `tfw-root.crt` contains
+public X.509 certificate, both are in PEM format. Now, you could use
+theese files as follows:
```

Comment:
The file `tfw-root.key` contains the private key, and the file `tfw-root.crt` contains the public X.509 certificate. Both are in PEM format. These files are used in Tempesta configuration as follows: